service-principal/version.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = "~> 1.0"
+  required_providers {
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "~> 2.46"
+    }
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = "3.92.0"
```

Comment:
```suggestion
      version = "~≳ 3.92"
```

Reply:
good tip, for modules it should work great